docs/usage.md

```diff
@@ -201,7 +201,7 @@ const sequelize = new Sequelize('sqlite:relativePath/dbname.db')
 
 ### PostgreSQL
 
-The library for PostgreSQL is`pg@~3.6.0` You'll just need to define the dialect:
+The library for PostgreSQL is`pg@^6.1.0 || ^7.0.0` You'll just need to define the dialect:
```

Comment:
I am not sure how outdated this doc is - maybe you could find out the minimal version that passes the tests? Does `pg@3.6.0` even work?